contracts/stellar-multicall/Cargo.toml

```diff
@@ -15,6 +15,10 @@ stellar-axelar-std = { workspace = true }
 goldie = { workspace = true }
 soroban-sdk = { workspace = true, features = ["testutils"] }
 stellar-axelar-std = { workspace = true, features = ["testutils"] }
+stellar-axelar-operators = { workspace = true }
+stellar-interchain-token = { workspace = true, features = ["testutils"] }
+soroban-token-sdk = { workspace = true }
+stellar-axelar-gas-service = { workspace = true, features = ["testutils"]}
```

Comment:
we should use test contracts for unit testing. what's the reason for using gas service etc.? The unit test should do the minimal necessary to make the test work. Also, if we make changes to the gas service, suddenly the multicall tests can break which is unexpected.